addons/uh60_ui/initSettings.sqf

```diff
@@ -19,3 +19,13 @@
     nil,
     {}
 ] call CBA_Settings_fnc_init;
+
+[
+    "vtx_uh60m_trackIR_interaction_systemChat",
```

Comment:
Can you change this to `vtx_uh60_hui_showDebugMessages`
The reason for this is because we're defining this variable in the uh60_ui pbo, so using that prefix allows us to know what module is providing this variable. All of our variables and functions are prefixed with the name of the module it's in